ykrt/src/location.rs

```diff
@@ -263,6 +268,7 @@ impl Location {
     /// Try obtaining the lock, returning the new `State` if successful.
     pub(super) fn try_lock(&self) -> Result<LocationInner, ()> {
         let mut ls = self.load(Ordering::Relaxed);
+        // FIXME: this could be in the counting state
```

Comment:
What is the consequence of this though? Can we add some more detail?

Reply:
This is a reminder for another PR that I just happened to spot while here.

Reply:
The comment is complete IMHO. I don't think we should obsess about this: it's a bug, there's a reminder, it's not related to this PR, best to move on.